manager/integration/tests/test_basic.py

```diff
@@ -2703,13 +2703,11 @@ def test_dr_volume_with_last_backup_deletion(
 
     md5sum1 = get_pod_data_md5sum(core_api, dr_pod_name, data_path1)
     assert std_md5sum1 == md5sum1
-    md5sum2 = get_pod_data_md5sum(core_api, dr_pod_name, data_path2)
-    assert std_md5sum2 == md5sum2
+
+    # Since backup#2 is newer than backup#1 and we deleted backup#2
```

Comment:
`For DR volume, the requested backup restore is backup1 and the last restored backup is backup2 now. Since the backup2 is gone, the DR volume will automatically fall back to do full restore for backup1.`

Reply:
1. Set a random backupstore.
    2. Create a volume, then create the corresponding PV, PVC and Pod.
    3. Write data to the pod volume and get the md5sum
       after the pod running.
    4. Create the 1st backup.
    5. Create two DR volumes from the backup.
    6. Wait for the DR volumes restore complete.
    7. Write data to the original volume then create the 2nd backup.
    8. Wait for the DR volumes incremental restore complete.
    9. Delete the 2nd backup.
    10. Verify the `lastBackup == 1st backup` for 2 DR volumes and original volume.
    11. Activate the DR volume 1 and wait for it complete.
    12. Create PV/PVC/Pod for the activated volume 1.
    13. Validate the volume content.
    14. Delete the 1st backup.
    15. Verify the `lastBackup == ""` for DR volume 2 and original volume.
    16. Activate the DR volume 2 and wait for it complete.
    17. Create PV/PVC/Pod for the activated volume 2.
    18. Validate the volume content, should be backup 1.